src/main/java/com/exactpro/epfast/template/CopyOperator.java

```diff
@@ -0,0 +1,9 @@
+package com.exactpro.epfast.template;
+
+public interface CopyOperator extends FieldOperator {
+
+    Dictionary getDictionary();
+
+    String getKey();
```

Comment:
return type for all `getKey()` methods should be `IdentityRef`. Rename to `getDictionaryKey`